.github/workflows/update_flake_lock.yml

```diff
@@ -0,0 +1,21 @@
+name: Update Flake Lock File
+on:
+  workflow_dispatch: # allows manual triggering
+  schedule:
+    - cron: '0 0 * * 0' # runs weekly on Sunday at 00:00
+
+jobs:
+  lockfile:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Install Nix
+        uses: DeterminateSystems/nix-installer-action@v5
+      - name: Update flake.lock
```

Comment:
Will this PR be skipped if no changes are needed to the flake.lock file?

Reply:
Yes

Reply:
"skipped" meaning not being open, then yes

Reply:
However since we are following `nixpkgs` unstable we might get PRs every week.
I will try to change this to `23-05`.

Reply:
Done in 6ba299accd547f36775fcdaadfb6359f77306d53